test/celeritas/ext/Vecgeom.test.cc

```diff
@@ -464,7 +471,7 @@ TEST_F(SolidsTest, trace)
         static real_type const expected_distances[] = {20,
                                                        95,
                                                        20,
-                                                       115,
+                                                       125,
```

Comment:
Do you know why the results changed when tracking through polycones?

Reply:
No... this seems to resolve the difference vs Geant4 I spotted in #759 (https://github.com/celeritas-project/celeritas/commit/ff926a41c95fb66b8fbb9b8a5ad37e808c7ec697) so I guess VecGeom changes behavior based on some edge condition?